packages/react/src/primitives/types/style.ts

```diff
@@ -178,8 +178,8 @@ export const ComponentPropsToStylePropsMap: ComponentPropToStyleProp = {
   marginBlockEnd: 'marginBlockEnd',
   marginBlockStart: 'marginBlockStart',
   marginBottom: 'marginBlockEnd',
-  marginInlineEnd: 'marginBlockEnd',
-  marginInlineStart: 'marginBlockStart',
+  marginInlineEnd: 'marginInlineEnd',
```

Comment:
ooops 😈

Reply:
Good catch

Reply:
Yeah good catch. That one's my fault, apologies!